labs/yocto-advanced-configuration/yocto-advanced-configuration.tex

```diff
@@ -17,7 +17,8 @@ \section{Set up the Ethernet communication and NFS on the board}
 First we need to set the kernel boot arguments U-Boot will pass to the
 Linux kernel at boot time:
 \begin{verbatim}
-setenv bootargs 'console=ttyS0,115200 root=/dev/nfs rw nfsroot=192.168.0.1:/nfs
+setenv bootargs 'console=ttyS0,115200 root=/dev/nfs rw
+  nfsroot=192.168.0.1:/nfs,nfsvers=3
```

Comment:
The tcp option should probably be added too because distributions are starting to remove udp support.

Reply:
Then, we should do it for all our labs, through a separate commit.

Reply:
Agreeing with the "nfsvers=3" variant